syntax = "proto3";

package adderPlusUltra;

service AddTwoNumbersBatch {
  rpc AddTwoNumbersBatch (AddTwoNumbersBatchRequest) returns (AddTwoNumbersBatchResponse) {}
}

message AddTwoNumbersBatchRequest {
  string s = 1;
}

message AddTwoNumbersBatchResponse {
  repeated int32 res = 2;
}